examples/2d/elixir_advection_basic.jl

```diff
@@ -6,32 +6,23 @@ using Trixi
 # semidiscretization of the linear advection equation
 
 advectionvelocity = (1.0, 1.0)
-# advectionvelocity = (0.2, -0.3)
 equations = LinearScalarAdvectionEquation2D(advectionvelocity)
 
-initial_condition = initial_condition_convergence_test
+solver = DGSEM(3,                   # polynomial degree
+               flux_lax_friedrichs) # surface flux function
 
-# you can either use a single function to impose the BCs weakly in all
-# 1*ndims == 2 directions or you can pass a tuple containing BCs for
-# each direction
-# Note: "boundary_condition_periodic" indicates that it is a periodic boundary and can be omitted on
-#       fully periodic domains. Here, however, it is included to allow easy override during testing
-boundary_conditions = boundary_condition_periodic
-
-surface_flux = flux_lax_friedrichs
-polydeg = 3
-solver = DGSEM(polydeg, surface_flux)
-
-coordinates_min = (-1, -1)
-coordinates_max = ( 1,  1)
-mesh = TreeMesh(coordinates_min, coordinates_max,
+coordinates_min = 
+coordinates_max = 
+mesh = TreeMesh((-1, -1), # minimum coordinates (min(x), min(y))
+                ( 1,  1), # maximum coordinates (max(x), max(y))
                 initial_refinement_level=4,
-                n_cells_max=30_000,
-                periodicity=true)
+                n_cells_max=30_000)
```

Comment:
Sounds good - I'll do it this way.

Reply:
The original idea was to make the number of lines as low as possible. If you introduce an additional variable (even temporary), it may give the hint to a completely new user that there is something special about this property. Also, otherwise the basic and the extended version only differ by 3-4 lines, and I almost thought "what's the point?". But your style is more readable, I agree.